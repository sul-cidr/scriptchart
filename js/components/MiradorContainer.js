import React from "react";

import MiradorViewer from './MiradorViewer';

const MiradorContainer = () => {
  return <MiradorViewer />;
};

export default MiradorContainer;
