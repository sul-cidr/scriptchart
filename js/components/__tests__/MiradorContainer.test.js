/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MiradorContainer from '../MiradorContainer';


describe('MiradorContainer test', () => {
  it('MiradorContainer should match snapshot', () => {
    const wrapper = shallow(<MiradorContainer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
