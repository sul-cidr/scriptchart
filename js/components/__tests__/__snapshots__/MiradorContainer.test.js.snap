// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MiradorContainer test MiradorContainer should match snapshot 1`] = `<MiradorViewer />`;
